examples/siro_sandbox/README.md

```diff
@@ -45,81 +33,18 @@ Build interactive desktop and VR applications that enable human-in-the-loop eval
 
 Check out our example HITL applications [here](../hitl/).
 
-All of these applications share core functionality provided by the HITL framework, including:
-* loading and simulating Habitat environments in real time
-* simulating virtual agents including policy inference
-* graphical user interfaces (GUIs) to enable real humans (users) to interact with these virtual agents
-
-To create a new HITL application, we recommend starting by copy-pasting one of the application folders like `hitl/pick_throw_vr/` into your own git repository, for example `my_repo/my_pick_throw_vr/`.
+Use these as a reference to create your own HITL application. we recommend starting by copy-pasting one of the example application folders like `hitl/pick_throw_vr/` into your own git repository, for example `my_repo/my_pick_throw_vr/`.
 
 ## VR HITL applications
 
 The HITL framework can be used to build desktop applications (controlled with keyboard/mouse) as well as **VR** applications. See our [Pick_throw_vr](../hitl/pick_throw_vr/README.md) example application.
 
+## Configuration
 
-## Workaround for poor runtime perf on slower machines
```

Comment:
All of this command-line documentation has been moved to hitl_defaults.yaml (reworded in some cases).